spec/system/reports/itemized_distributions_spec.rb

```diff
@@ -0,0 +1,29 @@
+RSpec.describe "Report Itemized Distributions", type: :system, js: true do
```

Comment:
If the spec is not interactive (i.e. not clicking something) there is no reason for it to be a system spec - it can be a much lower-weight request spec.